eng/InternalTools.props

```diff
@@ -1,6 +1,6 @@
-<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project>
-   
+    
```

Comment:
```suggestion
```

Reply:
Sigh, I was trying so hard not to mess with trailing spaces that I introduced one. Now to find out why VSCode wasn't telling me that happened.